src/frontend/src/components/public/PIASubHeader/index.tsx

```diff
@@ -229,27 +233,13 @@ function PIASubHeader({
   };
   const showSubmitButton = () => {
     const owner = getGUID() === pia.createdByGuid ? true : false;
```

Comment:
```suggestion
    const owner = getGUID() === pia.createdByGuid ?? false;
```
Do you need the ternary because shouldn't it only return true or false, and in the chance it returns undefined or null you're safe in this updated version as it will default to false.